ili2ora/src/ch/ehi/ili2ora/converter/OracleSpatialColumnConverter.java

```diff
@@ -44,26 +46,33 @@ public String getInsertValueWrapperMultiPolyline(String wkfValue,int srid) {
         return "ILI2ORA_SDO_GEOMETRY(" + wkfValue + ", "+ Integer.toString(srid)  + ")";
```

Comment:
ILI2ORA_SDO_GEOMETRY function wraps [SDO_GEOMETRY Constructor](https://docs.oracle.com/database/121/SPATL/sdo_geometry-constructors.htm#SPATL501) because it fails when geometry is null. Therefore, ILI2ORA_SDO_GEOMETRY returns null if the input geometry is null. Otherwise, it calls SDO_GEOMETRY Constructor.

ILI2ORA_SDO_GEOMETRY was added to ili2ora (#244) in visitSchemaBegin (GeneratorOracleSpatial) method.
```suggestion
    stmt += getIndent() + "CREATE OR REPLACE FUNCTION " + strWrapperFunction + "(geom_input BLOB, srid NUMBER)" + newline();
```
Do I move that code to fromIliInit (OracleCustomStrategy)?

Reply:
yes please move. The extensions of ch.ehi.sqlgen.generator.Generator (GeneratorOracleSpatial) should work independent of ili2db.

Reply:
I made the changes